### DISCUSSION TITLE
ondemand access without memcpy

### DISCUSSION THREAD

**Discussion by valkmit** [simdjson/simdjson#1420] [Q&A] [answered]
Hi all,

I noticed that the standard simdjson parser contains examples on how to parse without incurring a copy into a padded string. Is there an equivalent for the ondemand API?

Thanks in advance

**Comment by jkeiser** [comment, marked as answer]
@valkmit missed this, sorry--there is no such API right now, but I think we need to add one. Filed #1433 for it, thanks!

**Reply by jkeiser** [reply]
Note this will be fixed in the next version (see #1473).

**Reply by jha** [reply]
This is amazing work, thank you!

> Note this will be fixed in the next version (see #1473).
